src/style.css

```diff
@@ -0,0 +1,34 @@
+body,
+html {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-around;
+  width: 100px;
+}
+
+.stars__star {
+  --size: 16px;
+
+  width: var(--size);
+  height: var(--size);
+}
+
+.stars--0 > *,
+.stars--1 > :nth-child(n + 2),
+.stars--2 > :nth-child(n + 3),
+.stars--3 > :nth-child(n + 4),
+.stars--4 > :nth-child(5) {
+  background: url(./images/star.svg) left no-repeat;
+}
```

Comment:
set it as default to the stars__star

Reply:
Okey. But redesign one place is good?
I mean that in .stars__star I create all star noactive, and then part of then, again, recreate.